### PR TITLE
Add a note indicating Kibana will not recognize monitoring data unless it has something from the active master

### PR DESCRIPTION
**Pull request by PhaedrusTheGreek**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by PhaedrusTheGreek** [comment]
An alternative to this docs change is [this enhancement](https://github.com/elastic/kibana/issues/76389)

File: docs/reference/monitoring/configuring-metricbeat.asciidoc [hunk 1]
```diff
@@ -43,7 +43,8 @@ For more information, see <<monitoring-settings>> and <<cluster-update-settings>
 --
 
 . {metricbeat-ref}/metricbeat-installation-configuration.html[Install {metricbeat}] on each
-{es} node in the production cluster.
+{es} node in the production cluster (Monitoring data must be collected from at least the currently 
+active master node in order for Kibana to display it)
```
[the remark this answers is not in the document]
Comment (PhaedrusTheGreek) [reply docs/reference/monitoring/configuring-metricbeat.asciidoc:47]: Maybe so.  The documentation already states that monitoring agent must be installed on ALL nodes in the cluster - perhaps we should make that requirement more prominent?

[the remark this answers is not in the document]
Comment (jakelandis) [reply docs/reference/monitoring/configuring-metricbeat.asciidoc:47]: Maybe `...{es} node in the production cluster. Failure to install on each node may result in incomplete or missing results.` ?

**Review by lcawl — APPROVED** [review]
LGTM, thanks for doing this!

**Pull request merged**